status-im/status-react
# Added swipe to reply to chat message gesture

[Fixes #12361 ] Swipe-to-reply gesture

### Summary
When a user swipes on a textual chat message, the reply input will be launched and populated with the message being replied to. The swiped message will also show a spring animation.

### Review notes
This functionality is only on textual chat message. I didn't find images on public chats to test the feature on images.

### Testing notes
- Click on any public chat
- Swipe on a text message 
- See that the reply input has been populated and the swiped message has a spring animation for visual feedback upon  swiping
#### Platforms


- Android
- iOS
- macOS
- Linux
- Windows

#### Areas that maybe impacted


##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test


- Open Status
- ...
- Step 3, etc.



status: ready
